solutions/devsprint-pedro-alvarez-2/FinanceApp/Screens/Commons/CodeBaseView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  CodeBaseView.swift
+//  Pods
+//
+//  Created by Pedro Alvarez on 08/12/22.
+//
+import UIKit
+
+public class CodeBaseView<ViewModel>: UIView {
+  open var hierarchies: [HierarchyRelation] { [] }
+  
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+    setupHierarchy()
```

Comment:
opa @GusRigor, acha que seria interessante criar uma função chamada **setup()** onde teria esse dois métodos ? E no caso, dentro do init chamariamos apenas o **setup()**
```suggestion
func setup() {
    setupHierarchy()
    setupConstraints()
}
```

Reply:
Alterado.